streams/src/test/java/org/apache/kafka/streams/kstream/internals/TimeWindowTest.java

```diff
@@ -35,11 +35,6 @@
     private final TimeWindow window = new TimeWindow(start, end);
     private final SessionWindow sessionWindow = new SessionWindow(start, end);
 
-    @Test
-    public void endMustBeLargerThanStart() {
-        assertThrows(IllegalArgumentException.class, () -> new TimeWindow(start, start));
-    }
```

Comment:
Temporarily remove this test, and will be added back after KAFKA-13145 implements (renaming `TimeWindow` story)

Reply:
Why do we need to remove this temporarily?